source/_docs/configuration.markdown

```diff
@@ -28,6 +28,16 @@ If you run into trouble while configuring Home Assistant, have a look at the [co
 
 </div>
 
+## Editing `configuration.yaml`
+
+There are many ways you can edit `configuration.yaml`. Here are three options to get you started:
+
+The most basic is to use SSH to connect to the system (you may need to install the SSH add-on) and then use `nano` (or `vim`) to edit the file.
```

Comment:
Should we really list this suggestion for the first one? I can see my dad already trying this... 😂

Reply:
Moved to last